src/spyglass/lfp/analysis/v1/lfp_band.py

```diff
@@ -333,36 +333,16 @@ def make(self, key):
             + str(lfp_band_sampling_rate)
             + "Hz"
         )
-        tmp_valid_times = (
-            IntervalList
-            & {
+        interval_key = IntervalList().cautious_insert1(
+            key={
                 "nwb_file_name": key["nwb_file_name"],
                 "interval_list_name": key["interval_list_name"],
-            }
-        ).fetch("valid_times")
-        if len(tmp_valid_times) == 0:
-            lfp_band_valid_times = interval_list_censor(
```

Comment:
To avoid changing functionality, we should leave the `interval_list_censor` step in this make function